.github/workflows/main.yml

```diff
@@ -2,7 +2,6 @@ name: 🚀
 
 on:
   push: { branches: [main] }
-  pull_request: {}
```

Comment:
We should probably keep this, since this is still giving half of our current contributors security to merge. Imo it might be worth merging with failed workflow with non-core-contributors open PRs.

Reply:
Resolving as this comment has somehow been duplicated and I'm dealing with the other copy